frontend/src/app/components/ExperimentRowCard.js

```diff
@@ -156,7 +155,7 @@ export default class ExperimentRowCard extends React.Component {
 
 ExperimentRowCard.propTypes = {
   experiment: React.PropTypes.object.isRequired,
-  hasAddon: React.PropTypes.bool.isRequired,
+  hasAddon: React.PropTypes.any,
```

Comment:
You're right, a second prop would be much clearer. This started out as a quick hack at the beginning of the branch when I didn't think there was any possibility that this would land. I'll try to switch it to another property.